### PR TITLE
feat: support and feedback modal with third party support

### PR DESCRIPTION
**Pull request by zackpollard**
Adds a support and feedback button to the nav bar which will show a modal containing links to relevant documentation and issue trackers. Third party packages can set environment vars to fill their own trackers and documentation links in there. 
image
image

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]

File: docker/docker-compose.dev.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ services:
       - ${UPLOAD_LOCATION}/photos/upload:/usr/src/app/upload/upload
       - /usr/src/app/node_modules
       - /etc/localtime:/etc/localtime:ro
+      - ${UPLOAD_LOCATION}/external:/usr/src/app/external
```
Comment (danieldietzler) [inline docker/docker-compose.dev.yml:27]: Is that supposed to be in this PR?

**Review by danieldietzler — APPROVED** [review]

**Comment by martabal** [comment]
That's nice! But should this be displayed for all users or only admins? IMO users should first report bugs to the admins of their Immich install

**Comment by jrasm91** [comment]
> That's nice! But should this be displayed for all users or only admins? IMO users should first report bugs to the admins of their Immich install

I don't think it needs to be a requirement that you are a user to see information about how to report bugs or get support.

File: docker/docker-compose.dev.yml (hunk 1 above)
Comment (zackpollard) [reply docker/docker-compose.dev.yml:27]: Nope, accidentally made it in with the other changes 👀

**Comment by alextran1502** [comment]
I think a better place for the button is at the lower left corner since we are tidying up the top nav bar. I can help with that if you'd like

**Comment by zackpollard** [comment]
Personally I like it up there and it is where Google has there's. If we keep putting more stuff in the sidebar that's going to just look messy instead.

**Pull request merged**